cosmos/config.py

```diff
@@ -70,6 +70,7 @@ class RenderConfig:
     emit_datasets: bool = True
     test_behavior: TestBehavior = TestBehavior.AFTER_EACH
     load_method: LoadMode = LoadMode.AUTOMATIC
+    invocation_mode: InvocationMode = InvocationMode.DBT_RUNNER
```

Comment:
shall we add this in docstring?

Reply:
Done in 622be6cd4281fc44e82578187affe2e3884f979f